include/flatcc/flatcc_rtconfig.h

```diff
@@ -63,6 +63,14 @@ extern "C" {
 #define FLATCC_TRACE_VERIFY 0
 #endif
 
+/*
+ * Some producers allow empty vectors to be misaligned.
+ * The following setting will cause the verifier to check for an
+ * empty vector before checking alignment of the vector's elements.
+ */
```

Comment:
The following setting will not require the index 0 position to be element aligned iff the vector is empty, but it must still be aligned to the preceding size field.

(note: I am not 100% if C++ even aligns the size field correctly in this case, but I don't think I have seen evidence to the contrary. If the size field is not aligned, the problem is worse, both in verifier, and the reader also needs to be handled with more care).